spec/lib/elastic_model/integration/callbacks_with_parent_indices_spec.rb

```
@@ -72,6 +72,8 @@ def es_parent_id
           include ElasticModel::Callbacks
           belongs_to :parent_association
 
+          field :count, :type => Integer
+
```

It is used below though. Apparently Mongoid 3 is more forgiving about undefined fields?

(squashes an error under rails 4)